sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -101,4 +126,46 @@ export class AssessmentAESTController extends BaseController {
       true,
     );
   }
+
+  /**
+   * Triggers manual reassessment for an application.
+   * Application cannot be archived or in any of the statuses 'Cancelled', 'Overwritten' or 'Draft' and original assessment must be in completed status.
+   * @param payload request payload.
+   * @param applicationId application id.
+   * @returns id of the assessment created.
+   */
+  @Roles(Role.AESTManualTriggerReassessment)
+  @Post(":applicationId/manual-reassessment")
+  @ApiNotFoundResponse({ description: "Application id not found." })
+  @ApiUnprocessableEntityResponse({
+    description:
+      `Application original assessment expected to be '${StudentAssessmentStatus.Completed}' to allow manual reassessment or ` +
+      "application cannot have manual reassessment after being archived or " +
+      `application cannot have manual reassessment in any of the statuses: ${ApplicationStatus.Cancelled}, ${ApplicationStatus.Overwritten} or ${ApplicationStatus.Draft}.`,
+  })
+  async manualReassessment(
+    @Body() payload: ManualReassessmentAPIInDTO,
```

Comment:
`ManualReassessmentAPIInDTO` should be in the assessment DTO now. Does it make sense?